src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -388,10 +388,17 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('search.experimental.closedNotebookResults', "Show notebook editor rich content results for closed notebooks. Please refresh your search results after changing this setting."),
 			default: false
 		},
-		'search.experimental.autoAISearchResults': {
-			type: 'boolean',
-			description: nls.localize('search.experimental.autoAISearchResults', "Automatically request search results from the AI search provider"),
-			default: false
+		'search.searchView.autoSemanticSearchResults': {
```

Comment:
an idea
```suggestion
		'search.searchView.semanticSearchBehavior': {
```
based on these other ones
<img width="773" alt="image" src="https://github.com/user-attachments/assets/3baf9b43-6eac-4c75-8ac6-f49a7c4177dc" />

and then the enum is:
* `auto`
* `runOnEmpty`
* `manual`

Reply:
@TylerLeonhardt I like that idea a lot, what do you think of the latest commit?